assembly/activities/genMarkdownGuides.js

```diff
@@ -63,6 +63,8 @@ async function generateLangTabs(langtabs) {
     tabStr = `${tabStr}<TabItem value="${tab.lang}">\n\n`;
     if (tab.id == "none") {
       tabStr = `${tabStr}Content is currently unavailable.\n\n`;
+    } else if (tab.id == "na") {
+      tabStr = `${tabStr}The selected SDK does not currently support this feature.\n\n`;
```

Comment:
```suggestion
      tabStr = `${tabStr}Not applicable to this SDK.\n\n`;
```

don't want to ~imply that we may/will in future